### DISCUSSION TITLE
prompts inside tables

### DISCUSSION THREAD

**Discussion by Tylertron1998** [spectreconsole/spectre.console#271] [Q&A] [answered]
It would be super useful and nifty to have prompts inside tables - is this possible?

**Comment by patriksvensson** [comment, marked as answer]
@Tylertron1998 Alas, this is not possible, and I can't see us supporting this scenario without rewriting a lot of the internals of Spectre.Console. Frankly, this is by design.

**Reply by Tylertron1998** [reply]
That's very unfortunate.. it'd be nice to be able to layout every widget or feature (i.e. prompts, progress bars, etc) but I understand if it's rather complicated - thank you for your response!
